src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -96,6 +96,50 @@ internal IOperation GetOperationInternal(SyntaxNode node, CancellationToken canc
 
         protected abstract IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken);
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+        /// <summary>
+        /// Determines what type of conversion, if any, would be used if a given expression was
+        /// converted to a given type.  If isExplicitInSource is true, the conversion produced is
+        /// that which would be used if the conversion were done for a cast expression.
+        /// </summary>
+        public ConversionInfo ClassifyConversionInfo(SyntaxNode expression, ITypeSymbol destination, bool isExplicitInSource = false)
```

Comment:
>ClassifyConversionInfo [](start = 30, length = 22)

GetConversionInfo? Also, should this API take CancellationToken? It looks like GetConversion APIs take it.